src/plays/quizeo/src/Quizeo.jsx

```diff
@@ -0,0 +1,50 @@
+import { getPlayById } from 'meta/play-meta-util';
+
+import Header from './components/header/header.component';
+import Home from './pages/homePage/homePage.component';
+import MoviePage from './pages/moviePage/moviePage.component';
+import { Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import {store, persistor} from './redux/store'
+import { PersistGate } from 'redux-persist/integration/react';
+
+import './quizeo.css'
+import './App.css';
+
+
+
+import PlayHeader from 'common/playlists/PlayHeader';
+
+function Quizeo(props) {
+  // Do not remove the below lines. 
+  // The following code is to fetch the current play from the URL
+  const { id } = props;
+  const play = getPlayById(id);
+
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={play} />
+        <div className="play-details-body">
+        {/* Your Code Starts Here */}
+        <Provider store = {store}>
+            <PersistGate persistor={persistor}>
+              <div className="App">
+                <Header/>
+                <Routes>
+                  <Route exact path="/" element={<Home/>}/> 
+                  <Route exact path="/plays/quizeo/movie/:categoryId" element={<MoviePage/>}></Route>
```

Comment:
The framework has the route definition, you may have to define the route there. Routedef jsx file.

Reply:
Done

Reply:
Ok, also pull your branch from base main if not already.

I see you use Tailwind.  Could you please take look into the discussion going on for using Tailwind at the framework level? There are some upcoming plans, until that materialize, can we make your play with raw CSS?

Reply:
Well, I completed the play with tailwind. Is there no way to keep it? or else I'll change it to raw css.

Reply:
The way I am thinking is, that the framework should be independent of the dependencies of plays. Hence I will start working on creating each of the plays as the independent modules with its own package JSON, style system etc. That's when a play creator is free to use whatever they want. This change will start next week and may take a while. So if you want to submit the play sooner as it is done, and if writing raw CSS is not much, I will suggest going that way.

Otherwise, need to wait a while to get those changes done first. What say?

Reply:
Sure, no issue I will make the play to raw css and create a pull request.

Reply:
Are you not changing in the same PR? Btw, this is a great app. Looking forward to get it.

It should be of ADVANCED level.

Reply:
I'll make changes and commit to the same PR as soon as possible.